src/demo/components/counter-element.ts

```diff
@@ -56,9 +56,12 @@ class CounterElement extends HTMLElement {
   }
 
   _show() {
-    this.shadowRoot.getElementById('clicksSpan').textContent = this.clicks;
-    this.shadowRoot.getElementById('valueSpan').textContent = this.value;
+    this.shadowRoot!.getElementById('clicksSpan')!.textContent = this.clicks.toString();
```

Comment:
Can avoid the need for the exclamation points by declaring that the shadow root is a property that is not undefined. i.e. writing:
```typescript
  shadowRoot!: ShadowRoot;
```

with the other class properties